src/claim/Claim.ts

```diff
@@ -65,7 +65,12 @@ export default class Claim implements IClaim {
     if (input.contents !== undefined) {
       Object.keys(input.contents).forEach(key => {
         // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        if (typeof input.contents![key] !== 'string' && 'number' && 'boolean') {
+        const temp = input.contents![key]
+        if (
+          typeof temp !== 'number' &&
+          typeof temp !== 'boolean' &&
+          typeof temp !== 'string'
+        ) {
           throw new Error('Claim contents malformed')
         }
```

Comment:
calling `typeof temp` 3 times seems unnecessary, why not 
```suggestion
        const temp = typeof input.contents![key]
        if (
          temp !== 'number' &&
          temp !== 'boolean' &&
          temp !== 'string'
        ) {
          throw new Error('Claim contents malformed')
        }
```
or even better
```suggestion
         switch (typeof input.contents![key]) {
          case 'string':
          case 'number':
          case 'boolean':
              break;
          default:
            throw new Error('Claim contents malformed');
        }
```

Reply:
or maybe 
```ts
      Object.values(input.contents).forEach(value => {
        if (!['string', 'number', 'boolean'].includes(typeof value)) {
          throw new Error('Claim contents malformed')
        }
      })
```
if that is faster?

Reply:
Nice suggestions, settled for Object.entries, as it combines both functionalities, and we are able to check whether each key has a value without some non-null assertions